File: Implementação/Projetos/api-informacoes-cidadao/src/main/java/com/poctcc/api/model/PessoaJuridica.java
package com.poctcc.api.model;

import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@EqualsAndHashCode(onlyExplicitlyIncluded = true, callSuper = true)
public class PessoaJuridica extends Pessoa {

	private static final long serialVersionUID = 1L;

	private String cnpj;

	/**
	 * Está sendo considerado a Incrição Estadual no Formato de Minas Gerais.
	 * Format: "###.###.###/####"
	 */
	private String InscricaoEstadual;

	private String razaoSocial;

	private String nomeFantasia;

}

File: Implementação/Front-End/portal-cidadao/src/app/models/PessoaJuridica.ts
import { Pessoa } from "./Pessoa";

export class PessoaJuridica extends Pessoa {
    public cnpj: string;

	/**
	 * Está sendo considerado a Incrição Estadual no Formato de Minas Gerais.
	 * Format: "###.###.###/####"
	 */
	public inscricaoEstadual: string;

	public razaoSocial: string;

	public nomeFantasia: string;
}